seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/SaveModePlaceHolderEnum.java

```diff
@@ -17,13 +17,24 @@
 
 package org.apache.seatunnel.api.sink;
 
-public class SaveModeConstants {
+public enum SaveModePlaceHolderEnum {
+    ROWTYPE_PRIMARY_KEY("primaryKeys"),
+    ROWTYPE_UNIQUE_KEY("uniqueKeys"),
+    ROWTYPE_FIELDS("fields"),
+    TABLE_NAME("tableName"),
+    DATABASE("database");
```

Comment:
Please do not change the key value. We need to ensure backward compatibility

Reply:
> Please do not change the key value. We need to ensure backward compatibility

There are already methods in the class to get the placeholder, the enumeration names are the same as before, and the keyValue here is used for other purposes.